### PR TITLE
Quick start - downloading files

### PR DESCRIPTION
**Pull request by wwysoc2**

### CODE REVIEW

File: docs/Data_Portal/Users_Guide/quick_start.md [hunk 1]
```diff
@@ -11,23 +13,74 @@ __Step 1:__ Go to the Cohort Builder at the top left of the Data Portal.
...
+__Step 3:__ In the Repository tool, the files will only be those that are associated with the cases in your active cohort. Narrow down your file selection by filtering them using the panel on the left.
+
+<details>
+  <summary><i>Click to Expand/Collapse Animation</i></summary>
+[![Animation of Filtering the Repository](images/agifs/SelectFiltersRepo-animation.gif)](images/agifs/SelectFiltersRepo-animation.gif "Click to see the full image.")
+</details>
+
+---
+
+__Step 4:__ When you are done filtering.  Add the files to your cart.  Then go to the cart.
```
Comment (clsackett) [inline docs/Data_Portal/Users_Guide/quick_start.md:72]: Maybe a typo (?) - "When you are done filtering, add the files to your cart."

File: docs/Data_Portal/Users_Guide/quick_start.md [hunk 2]
```diff
@@ -11,23 +13,74 @@ __Step 1:__ Go to the Cohort Builder at the top left of the Data Portal.
...
+
+<details>
+  <summary><i>Click to Expand/Collapse Animation</i></summary>
+[![Animation of Adding Files to and Browsing to the Cart](images/agifs/AddFilesCart-animation.gif)](images/agifs/AddFilesCart-animation.gif "Click to see the full image.")
+</details>
+
+---
+
+__Step 5:__ The files in the cart can be downloaded directly from the browser or a manifest can be downloaded and passed to the GDC Data Transfer Tool.
```
Comment (clsackett) [inline docs/Data_Portal/Users_Guide/quick_start.md:81]: Would adding a link to the GDC DTT documentation be helpful here?

**Review by clsackett — APPROVED** [review]
Two small comments, otherwise LGTM!

File: docs/Data_Portal/Users_Guide/quick_start.md [hunk 3]
```diff
@@ -11,23 +13,74 @@ __Step 1:__ Go to the Cohort Builder at the top left of the Data Portal.
...
 __Step 4:__ The cohort you created is now your active cohort.  Go to the Analysis Center at the top left of the Data Portal.  Choose the tool you would like to use from the list.  The analysis will apply to the data from your cohort.
 
 <details>
   <summary><i>Click to Expand/Collapse Animation</i></summary>
 [![Animation of Browsing to the Analysis Center](images/agifs/GotoAC-animation.gif)](images/agifs/GotoAC-animation.gif "Click to see the full image.")
 </details>
+
+## Downloading Files
+
+__Step 1:__ Have an active cohort selected.  If you do not have an active cohort, [create one](#building-and-analyzing-a-cohort).
```
Comment (wteouchicago) [inline docs/Data_Portal/Users_Guide/quick_start.md:45]: I don't think it's possible not to have an active cohort :) ... maybe rephrase to "...active cohort containing cases of interest...", "...cases whose files you want to download...", etc? 🤷‍♀️

File: docs/Data_Portal/Users_Guide/quick_start.md (hunk 2 above)
Comment (wteouchicago) [reply docs/Data_Portal/Users_Guide/quick_start.md:81]: +1

File: docs/Data_Portal/Users_Guide/quick_start.md (hunk 1 above)
Comment (wwysoc2) [reply docs/Data_Portal/Users_Guide/quick_start.md:72]: 100% a typo, will fix with next commit.

File: docs/Data_Portal/Users_Guide/quick_start.md (hunk 2 above)
Comment (wwysoc2) [reply docs/Data_Portal/Users_Guide/quick_start.md:81]: I thought the same thing, then I forgot, will add it in.

**Comment by wwysoc2** [comment]
I'm going to merge this to verify it builds correctly.  Will probably bother you both with more PRs on this page between now and soft launch.
